cosmos.gemspec

```diff
@@ -100,6 +100,7 @@ spec = Gem::Specification.new do |s|
   s.add_runtime_dependency 'stopgap_13632', '~> 1.1.1' if RUBY_ENGINE == 'ruby' and %w(2.2.7 2.2.8 2.3.4 2.4.1).include? RUBY_VERSION  # MRI Only
 
   # Development Dependencies
+  s.add_development_dependency 'diff-lcs', '~> 1.3' if RUBY_ENGINE == 'ruby' # Get latest for MRI
```

Comment:
1.3 has specific fixes for warnings in Ruby 2.4 but it has dependency conflicts on JRuby